plasmapy/particles/decorators.py

```diff
@@ -240,10 +254,10 @@ def annotations(self) -> dict[str, Any]:
         -------
         `dict` of `str` to `object`
         """
-        return self._data.get("annotations")
+        return self._data.get("annotations")  # type: ignore[return-value]
```

Comment:
I didn't want to bother with fixing this since there's a new way to get annotations in Python 3.10, and we're dropping Python 3.9 support soon anyway.